src/main/java/net/dv8tion/jda/core/entities/impl/GuildImpl.java

```diff
@@ -112,6 +115,46 @@ public VoiceChannel getAfkChannel()
     }
 
     @Override
+    public RestAction<List<Webhook>> getWebhooks()
+    {
+        if (!getSelfMember().hasPermission(Permission.MANAGE_WEBHOOKS))
+            throw new PermissionException(Permission.MANAGE_WEBHOOKS);
+
+        Route.CompiledRoute route = Route.Guilds.GET_WEBHOOKS.compile(id);
+
+        return new RestAction<List<Webhook>>(api, route, null)
+        {
+            @Override
+            protected void handleResponse(Response response, Request request)
+            {
+                if (!response.isOk())
+                {
+                    request.onFailure(response);
+                    return;
+                }
+
+                List<Webhook> webhooks = new LinkedList<>();
+                JSONArray array = response.getArray();
+                EntityBuilder builder = EntityBuilder.get(getJDA());
+
+                for (Object object : array)
+                {
+                    try
+                    {
+                        webhooks.add(builder.createWebhook((JSONObject) object));
+                    }
+                    catch (JSONException | NullPointerException e)
+                    {
+                        JDAImpl.LOG.log(e);
```

Comment:
It appears that this is where you are using the NullPointerException from EntityBuilder#createWebhook, however, like i said above, you need to provide a description of the exception in the exception.